tests/Authentication/MagicLinkTest.php

```diff
@@ -114,6 +114,9 @@ public function testMagicLinkVerifyExpired(): void
 
         $result->assertRedirectTo(route_to('magic-link'));
         $result->assertSessionHas('error', lang('Auth.magicLinkExpired'));
+
+        // It should have set temp session var
```

Comment:
```suggestion
        // It should not have set temp session var
```

Reply:
@MGatner I'm fine with both if we want that. My initial instinct was an event, but you can't redirect from an event so you would have to set a session variable or something anyway to know you needed to do something so this saves that step. I couldn't really think of too many actions that would need to be done from an event at the time.

Reply:
I don't think the Event hurts anything. A dev could actually throw a `RedirectException` from the Event trigger to force a redirect - not the best, but it might actually be an appropriate use here.

Reply:
@MGatner added the event.